de-lib/src/main/java/org/iplantc/de/client/util/AnalysisSubmissionUtil.java

```diff
@@ -0,0 +1,105 @@
+package org.iplantc.de.client.util;
+
+import org.iplantc.de.client.models.apps.integration.AppTemplate;
+import org.iplantc.de.client.models.apps.integration.Argument;
+import org.iplantc.de.client.models.apps.integration.ArgumentGroup;
+import org.iplantc.de.client.models.apps.integration.ArgumentType;
+import org.iplantc.de.client.models.apps.integration.JobExecution;
+import org.iplantc.de.client.models.apps.integration.SelectionItem;
+import org.iplantc.de.client.models.apps.integration.SelectionItemGroup;
+
+import com.google.common.base.Strings;
+import com.google.web.bindery.autobean.shared.AutoBeanCodex;
+import com.google.web.bindery.autobean.shared.AutoBeanUtils;
+import com.google.web.bindery.autobean.shared.Splittable;
+import com.google.web.bindery.autobean.shared.impl.StringQuoter;
+
+
+public class AnalysisSubmissionUtil {
+
+    public static Splittable assembleLaunchAnalysisPayload(final AppTemplateUtils appTemplateUtils,
```

Comment:
🌟 Nice refactor!  I think eventually we might want to create models/autobeans for this stuff, but probably not for this PR

Reply:
I didn't create autobeans for this stuff because the splittable created here is going to be sent as body to an API call (and i didn't want to write more GWT code )